services/route_manager/src/main/java/com/futurewei/alcor/route/controller/RouteController.java

```diff
@@ -252,4 +251,43 @@ public ResponseId deleteRuleWithSubnetId(@PathVariable String subnetId, @PathVar
         return new ResponseId(routeId);
         
     }
+
+    @RequestMapping(
+            method = POST,
+            value = {"/routers/{routerId}/routetable/{routeTableId}"})
+    @ResponseStatus(HttpStatus.CREATED)
+    @DurationStatistics
+    public RouterWebJson createDefaultRouterForVpc(@PathVariable String routerId, @PathVariable String routeTableId, @RequestBody VpcWebJson resource) throws Exception {
+        RouteEntry routeEntry = null;
+        RouteTable routeTable = null;
+        Router router = null;
+
+        try {
+            RestPreconditionsUtil.verifyParameterNotNullorEmpty(routerId);
+            RestPreconditionsUtil.verifyParameterNotNullorEmpty(routeTableId);
+            VpcEntity vpcEntity = resource.getNetwork();
+            RestPreconditionsUtil.verifyResourceNotNull(vpcEntity);
+
+            String id = UUID.randomUUID().toString();
+            String projectId = vpcEntity.getProjectId();
+            String destination = vpcEntity.getCidr();
+            String vpcId = vpcEntity.getId();
+
+            routeEntry = new RouteEntry(projectId, id, "default_route_rule", "",
+                    destination, RouteConstant.DEFAULT_TARGET, RouteConstant.DEFAULT_PRIORITY, routeTableId, null);
+            List<RouteEntry> routeEntries = new ArrayList<>();
+            routeEntries.add(routeEntry);
+            routeTable = new RouteTable(projectId, routerId, "default_route_table", "",routeEntries, RouteConstant.DEFAULT_ROUTE_TABLE_TYPE,vpcId);
+
+            router = new Router(projectId, routeTableId, "default_router", "", routeTable);
+
+            //this.routeDatabaseService.addRoute(routeEntity);
```

Comment:
Do we need to persist here?

Reply:
No ~ this is an API for previous test, which need to be deleted